common/pom.xml

```diff
@@ -56,7 +56,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <configuration>
-          <maxAllowedViolations>114</maxAllowedViolations>
+          <logViolationsToConsole>true</logViolationsToConsole>
```

Comment:
Which mvn goal do you use? I only see errors displayed when I explicitly use checkstyle:check. For the "verify" goal  I don't see individual errors printed, only the total number of errors.